bibtex/bib/quantum.bib

```diff
@@ -28,6 +17,70 @@ @Article{BB14
+  Timestamp                = {2015.07.24}
+}
+
+@Article{BBCS92,
+  Title                    = {{Practical Quantum Oblivious Transfer}},
+  Author                   = {Bennett, C H and Brassard, G and Cr\'{e}peau, C and Skubiszewska, M H},
+  Journal                  = {Lect. Notes Comput. Sci.},
+  Year                     = {1992},
+  Pages                    = {351--366},
+  Volume                   = {576},
+
+  Abstract                 = {We describe a protocol for quantum oblivious transfer, utilizing faint pulses Of polarized light, by which one of two mutually distrustful parties ('Alice'') transmits two one-bit messages in such a way that the other party (''Bob'') can choose which message he gets but cannot obtain information about both messages (he will learn his chosen bit's value with exponentially small error probability and may gain at most exponentially little information about the value of the other bit), and Alice will be entirely ignorant of which bit he received. Neither party can cheat (ie deviate from the protocol while appearing to follow it) in such a way as to obtain more information than what is given by the description of the protocol. Our protocol is easy to modify in order to implement the All-or-Nothing Disclosure of one out of two string messages, and it can be used to implement bit commitment and oblivious circuit evaluation without complexity-theoretic assumptions, in a way that remains secure even against cheaters that have unlimited computing power. Moreover, this protocol is practical in that it can be realized with available opto-electronic apparatus while being immune to any technologically feasible attack for the foreseeable future.},
+  Doi                      = {10.1007/3-540-46766-1\_29},
+  ISBN                     = {978-3-540-55188-1},
+  ISSN                     = {0302-9743},
+  Owner                    = {chris},
+  Timestamp                = {2015.08.20}
+}
+
+@Misc{BBE92,
+  Title                    = {{Quantum Cryptography}},
+
+  Author                   = {Bennett, Charles H. and Brassard, Gilles and Ekert, Artur K.},
+  Year                     = {1992},
+
+  Abstract                 = {The desire to communicate in total secrecy is probably$\backslash$nas old as humankind. Myriad codes and ciphers have been$\backslash$ndevised only to be broken by mathematicians. Quantum$\backslash$nphysics may finally give the communicating parties a$\backslash$ndecisive edge. Because observing a quantum phenomenon$\backslash$nperturbs that which is seen, any attempt at$\backslash$neavesdropping will alert the legitimate users.},
+  Booktitle                = {Sci. Am.},
+  Doi                      = {10.1038/scientificamerican1092-50},
+  ISSN                     = {0036-8733},
+  Number                   = {4},
+  Owner                    = {chris},
+  Pages                    = {50--57},
+  Timestamp                = {2015.08.20},
+  Volume                   = {267}
+}
+
+@Book{BBD09,
```

Comment:
it's already included. Strangely, it's at the end of the bib-file, I don't understand why...